Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Generated by https://smithery.ai. See: https://smithery.ai/docs/config#dockerfile
+# Use the official Node.js image as the base image
+FROM node:14-alpine AS builder
```

Comment:
Shouldn't this be a more modern version of node?

Reply:
Yes, we should use version 22